cmd/server/assets/email/default_invite.html

```diff
@@ -9,5 +9,5 @@
 {{.InviteLink}}
 
 Thanks,
-The encv team
+The ENCV team
```

Comment:
this needs to be configurable - ENCV is the URL in the US.

Ideally this would be realm specific

Reply:
The idea is to allow realms to override entirely. This is just the default